app/admin/dashboard.rb

```diff
@@ -62,5 +62,19 @@
         end
       end # column
     end # columns
+
+    columns do
+      column do
+        panel "Recent Withdrawal Requests" do
+          table_for Withdrawal.order('id desc').limit(10).each do |withdrawal|
```

Comment:
Create a scope which will return to you just the records that are not approved yet. http://guides.rubyonrails.org/active_record_querying.html#scopes